src/gs-shell.c

```diff
@@ -782,6 +788,37 @@ gs_shell_reload_cb (GsPluginLoader *plugin_loader, GsShell *shell)
 	}
 }
 
+static void
+gs_shell_reload_and_select_removable_media_category (GsShell *shell,
+						     gboolean reload_overview,
+						     gboolean select_category,
+						     const gchar *category_name)
```

Comment:
I guess this is well aligned but Github shows it badly? (Just making sure, because it's happened before)

Reply:
It seems to be a Builder bug. It looks fine in Builder (double-checked all the tabs and spacing) but it also shows mis-aligned in `gnome-terminal`, so I guess Builder is displaying it incorrectly for some reason.

Fixed locally.

Reply:
In Emacs it's usually pretty consistent, you can verify with it too if you want.